client/layout/index.jsx

```diff
@@ -167,6 +168,7 @@ Layout = React.createClass( {
 					isActive={ translator.isActivated() } />
 				{ this.renderPreview() }
 				{ config.isEnabled( 'happychat' ) && this.props.chatIsOpen && <AsyncLoad require="components/happychat" /> }
+				<AppBanner />
```

Comment:
Oh Interesting, I was thinking we would drop this in the 4 sections we'd like, so we don't need to look at the current route.